app/views/request/_incoming_correspondence.html.erb

```diff
@@ -23,6 +23,25 @@
                             :body => incoming_message.get_body_for_html_display(@collapse_quotes),
                             :attachments => incoming_message.get_attachments_for_display }  %>
 
+    <% if incoming_message.exemptions.any? %>
```

Comment:
Let's drop this from the PR as it's not intended to get merged

Reply:
Done